Jenkinsfile

```diff
@@ -139,48 +151,20 @@ podTemplate(name: 'fedora-atomic-inline', label: 'fedora-atomic-inline', cloud:
                         messageContent = ''
                         sendMessage(messageProperties, messageContent)
 
-                        // Provision of resources
-                        allocDuffy("${current_stage}")
-
-                        echo "Duffy Allocate ran for stage ${current_stage} with option ${env.DUFFY_OP}\r\n" +
-                                "ORIGIN_WORKSPACE=${env.ORIGIN_WORKSPACE}\r\n" +
-                                "ORIGIN_BUILD_TAG=${env.ORIGIN_BUILD_TAG}\r\n" +
-                                "ORIGIN_CLASS=${env.ORIGIN_CLASS}"
-
                         job_props = "${env.ORIGIN_WORKSPACE}/job.props"
                         job_props_groovy = "${env.ORIGIN_WORKSPACE}/job.groovy"
                         convertProps(job_props, job_props_groovy)
                         load(job_props_groovy)
 
-                        // Stage resources - RPM build system
-                        setupStage("${current_stage}")
-
-                        if (env.OSTREE_BRANCH == null) {
-                            env.OSTREE_BRANCH = ""
+                        // Call rpmbuild-test.sh
+                        // in rpmbuild container
+                        container('rpmbuild') {
+                            sh "fed_repo=${env.fed_repo} " +
+                               "fed_branch=${env.fed_branch} " +
+                               "fed_rev=${env.fed_rev} " +
+                               "/home/rpmbuild-test.sh"
                         }
 
-                        // Rsync Data
-                        writeFile file: "${env.ORIGIN_WORKSPACE}/task.env",
-                                text: "export JENKINS_JOB_NAME=\"${JOB_NAME}-${current_stage}\"\n" +
-                                        "export JENKINS_BUILD_TAG=\"${BUILD_TAG}-${current_stage}\"\n" +
-                                        "export OSTREE_BRANCH=\"${OSTREE_BRANCH}\"\n" +
-                                        "export fed_repo=\"${fed_repo}\"\n" +
-                                        "export fed_branch=\"${fed_branch}\"\n" +
-                                        "export fed_rev=\"${fed_rev}\"\n"
-                        rsyncResults("${current_stage}")
-
-                        def package_props = "${env.ORIGIN_WORKSPACE}/logs/package_props.txt"
```

Comment:
I assume there are some properties from the rpm build container that need to be exposed to the next stage?  How does that work in this model?